### PR TITLE
orwi init

### PR DESCRIPTION
**Pull request by semihaltintas**

### CODE REVIEW

**Comment by semihaltintas** [comment]
why rejected i dont understand

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
Because no one was there to approve the deployment. Kicked this off again.

**Comment by mcollina** [comment]
@semihaltintas can you remove the empty space at the top and the bottom?

**Comment by mcollina** [comment]
@Eomm can you take a look? Previews do not seem to be working anymore.

**Review by Eomm — COMMENTED** [review]
The SVG is not good for the website:

image

**Comment by semihaltintas** [comment]
ok i will fix it and push new svg.

**Comment by semihaltintas** [comment]
Is it good for website?

File: static/img/organisations/orwi.svg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg id="katman_1" data-name="katman 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 400 400">
+<svg id="katman_1" data-name="katman 1" xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 287.2 102.2">
```
Comment (Eomm) [inline static/img/organisations/orwi.svg]: I think we need the `width="287.2" height="102.2"` code here

Comment (semihaltintas) [reply static/img/organisations/orwi.svg]: fixed.

**Review by Eomm — APPROVED** [review]
image

**Pull request merged**